### ISSUE TITLE
Alt+f1 not passed to the Monaco editor

### ISSUE THREAD

**Issue by devil418** [notable/notable#631]
- **OS Version**: Windows 7 64 bit polish
- **Notable Version**: 1.4.0

### Current behavior
When editing a note, the Alt+f1 combination, supposed to make the Monaco editor accessible to screen readers, does not work. It's supposed to open a sort of pop up that tells us to press ctrl+e to enable accessibility, but it does not appear. The accessibility mode is necessary to use Monaco with a screen reader in any meaningful way. To reproduce, try using a screen reader [NVDA](nvaccess.org) on Windows, the built-in Voice Over, turned on by pressing Command+f5 on Mac. No screen reader that works with Electron exists on Linux yet. After turning it on, try focusing the edit field. You should notice that the Screen reader says something like "The editor is not accessible at this time. Press Alt+F1 for options.  edit  has auto complete  multi line". I'm not sure what the shortcut is on Mac, and I have no Mac to test this, it may be different.

### Expected behavior

Pressing Alt+F1 should open Monaco's accessibility settings pop up. Foe examples, see the Monaco version running on repl.it. It would be also appreciated if accessibility settings persisted across Notable's restarts, as turning them on every time would be quite annoying. VS Code does this properly, repl.it does not.

**Comment by fabiospampinato** [comment]
Accessibility support in monaco editor has been explicitly [turned off](https://github.com/notable/notable/blob/9817109d22d20a45532d10e08ddbd6216cbef12f/src/renderer/utils/monaco.ts#L19) because I didn't have the time to check how it worked, if it messed up other things in the process, and because other parts of Notable are probably not very accessible (since it was meant to be just an app for my personal needs).

I'll be happy to look into making the app more accessible in the future.
